internal/resources/templates/generic_results.go

```diff
@@ -24,11 +24,27 @@ const GenericResults = `
 			<div class="ui tabs divider"></div>
 		</div>
 		<div class="ui container">
+		<div class="ui grid">
+		<div class="column" style="width:65%">
 			<hr>
 			<div>
 				<pre>{{.Content}}</pre>
 			</div>
 		</div>
+		<div class="column" style="width:35%">
+			<div id="history">History:</div>
+			{{ range $val := .Results }}
+			<div>
+				<a href="{{$val.Href}}" alt="{{$val.Alt}}">
+					{{$val.Badge}}
+					<span class="date-tiny">{{$val.Text1}}</span>
```

Comment:
Merge both spans into one with padding: `<span class="tiny" style="padding-left:20px">{{$val.Text1}} {{$val.Text2}}</span>`, or add the padding to the css.